Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/settings/SettingsActivity.java

```diff
@@ -50,6 +50,22 @@ protected void onCreate(Bundle savedInstanceState)
 		Intent launcher = getIntent();
 		String gameID = launcher.getStringExtra(ARG_GAME_ID);
         MenuTag menuTag = (MenuTag) launcher.getSerializableExtra(ARG_MENU_TAG);
+
+		if (menuTag == MenuTag.GRAPHICS)
```

Comment:
I like the idea, but why not to cover all cases. Better have a map between the MenuTag and the equivalent title, this will element the need to add one if per each case and you can cover all cases available: Enhancements, Hacks, General, Interface, GameCube, Wii, Wiimote Extension, etc

Reply:
I agree. I first wanted to get feedback on using menu tags for dynamic SettingsActivity titles. I'm teaching myself Android and I haven't looked into map yet. Is there a particular type of map that you think would be best for this situation and a similar implementation elsewhere in Dolphin? I'll research map on Android.

Reply:
I'll look into that and mention you when it's updated. Thanks!

Reply:
https://docs.oracle.com/javase/8/docs/api/java/util/Map.html

Your code will look like this:
```
private static final Map<MenuTag, @StringRsc Integer> titles = new HashMap<>();
static {
  titles.put(MenuTag.GRAPHICS, R.string.grid_menu_graphics_settings);
  titles.put(MenuTag.GCPAD_TYPE, ...);
  // ...
}

// ... {
  if (titles.contains(menuTag)) {
    setTitle(titles.get(menuTag));
  }
```